### ISSUE TITLE
New Term - causeOfDeath

### ISSUE THREAD

**Issue by qgroom** [tdwg/dwc#521]
## New term

* Submitter: Quentin Groom, Toke Thomas Høye (@tokehoye), Lien Reyserhove (@LienReyserhove)
* Efficacy Justification (why is this term necessary?): The term dwc:vitality allows users to indicate whether an organism was dead when it was observed. Although the cause of death may often be unknown, in cases when it is identifiable, this information is invaluable for a wide range of studies. These include assessing the impact of invasive species, monitoring the spread of wildlife diseases, evaluating human activities' effects on wildlife, and more.
* Demand Justification (name at least two organisations that independently need this term): From the dwc:vitality term proposal the following organisations were mentioned, NBN Trust, Ocean Biodiversity Information System (OBIS), The Field Museum, Global Genome Biodiversity Network (GGBN), to those I add Meise Botanic Garden, The Research Institute for Nature and Forest (INBO) and Aarhus University.
The need for a causeOfDeath term has been thoroughly discussed by the "How did it die?" task group, as documented in issues[ #228](https://github.com/tdwg/dwc/issues/228) and[ #363](https://github.com/tdwg/dwc/issues/363). 
The issue 228 was closed when the term dwc:vitality was accepted into Darwin Core, yet the need for a causeOfDeath term persists. In my specific case, this term is crucial for a project that aims to identify roadkill animals using AI, but there are many other documented use cases. While there were discussions about creating an associated vocabulary, it was recognized that this would be an enormous task, and potentially overly restrictive for the term. Therefore, like many other terms in Darwin Core, it is likely more effective to recommend the use of a vocabulary without mandating it.
* Stability Justification (what concerns are there that this might affect existing implementations?): none
* Implications for dwciri: namespace (does this change affect a dwciri term version)?: no 

Proposed attributes of the new term:

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): causeOfDeath
* Term label (English, not normative): Cause Of Death
* Organized in Class (e.g., Occurrence, Event, Location, Taxon): Organism
* Definition of the term (normative): An indication of the known or suspected cause of death of a dwc:Organism. This may include natural causes (e.g., disease, predation), human-related activities (e.g., roadkill, pollution), or other environmental factors (e.g., extreme weather events).
* Usage comments (recommendations regarding content, etc., not normative): Intended to be used only when the value in dwc:vitality indicates that a dwc:Organism was dead at the moment of observation.
* Examples (not normative): trapped, poisoned, starved, drowned, shot, old age, roadkill, disease, herbicide, burned
* Refines (identifier of the broader term this term refines; normative): none
* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): none
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative): none

**Comment by qgroom** [comment]
Although the "How Did it Die" task group is not active there are some relevant notes on the webpage https://www.tdwg.org/community/osr/how-did-it-die/

**Comment by tokehoye** [comment]
Camera traps fitted to cars are beginning to emerge (e.g. https://doi.org/10.3390/s21186126) and pilot testing suggest that they can facilitate the observation of road kill animals. We are making such trials in EU-wide projects. To be able to make such data FAIR, it will be important to adapt metadata standards to these new sensor-based observations. Thanks to @qgroom for proposing this new term!

**Comment by gparosenberg** [comment]
I favor adding this term to the Darwin Core. I think we would need to distinguish observation from inference. Mollusk collections sometimes have clams killed by seagulls, by dropping them on rocks or roads. Sometimes death of the organism is directly observed, but more often it is inferred from the condition of the specimen.

**Comment by tmcelrath** [comment]
I favor this term as well. Would allow entomological collections to add things like "dead in a spider web" or "killed by a specific organism".

**Comment by gparosenberg** [comment]
I've added this term to our database and started trying to populate it. There's a lot of inference involved: for example, "ex pisces" is sometime stated in a record, so I've added for cause of death "eaten by fish".  That's a proximal reason; also mollusks are known to occasionally pass alive through the digestive tracts of fish and birds, so it's an inference that might at times be wrong. (The specimen might have been alive at the time it was collected.)

In the Eastern Seaboard TCN, we've added the vitality field and we also require a basis of inference of vitality status. The later can be reported as an "occurrenceRemark," which shows up a note in Invertebase (e.g,, https://invertebase.org/portal/collections/individual/index.php?occid=9197331&clid=0). It turns out that when cause of death is known or inferred, that would also serve as the basis of inference of vitality.

**Comment by tucotuco** [comment]
I think this term should be proposed to be organized in the Organism class rather than the Occurrence class. It is an Organism that dies, not an Occurrence of an Organism.

**Comment by qgroom** [comment]
> I think this term should be proposed to be organized in the Organism class rather than the Occurrence class. It is an Organism that dies, not an Occurrence of an Organism.

I don't really understand the distinction I think I was following the same logic as these terms - individualID, lifeStage, reproductiveCondition, sex and vitality

**Comment by tucotuco** [comment]
The terms lifeStage, reproductiveCondition, and vitality are all attributes of an organism at a place and time - things that can change from one Event (Occurrence) to another. The causeOfDeath term would be singularly tied to the Organism - it is not something that can change from one Event (Occurrence) to another.

The term individualID no longer exists in Darwin Core.

**Comment by qgroom** [comment]
That makes sense, thanks for the explanation.

BTW: individualID is still mentioned here https://dwc.tdwg.org/list/ I guess it needs correcting.

**Comment by tucotuco** [comment]
> BTW: individualID is still mentioned here https://dwc.tdwg.org/list/ I guess it needs correcting.

Good catch.  @baskaufs Something must be wrong with the maintenance scripts that produce the term list document. Thanks!

**Comment by baskaufs** [comment]
No this is not an error. The list of terms document contains every term that has ever been minted in the dwc: and dwciri: namespaces, including deprecated ones. See https://dwc.tdwg.org/list/#dwc_individualID, which contains the note "This term is deprecated and should no longer be used."

All of the terms ever minted need to be included in the list of terms, since when terms in those namespaces are dereferenced, they redirect to fragment identifiers in the list of terms document. If they were not included in the list of terms document, there would effectively be no way for someone to discover the meaning of the term by dereferencing its IRI: http://rs.tdwg.org/dwc/terms/individualID

**Comment by LienReyserhove** [comment]
Thank you @qgroom for proposing this term. I believe it will be a valuable addition to the DwC standard!

**Comment by tucotuco** [comment]
Thanks @baskaufs . Of course. And a relief.

**Comment by vijaybarve** [comment]
I favor the addition of this term, would be particularly useful for roadkill datasets and undesirable reasons like electrocution and wind mills etc.

**Comment by dbloom** [comment]
I agree that this would make a good addition. It will be of relevance to the recently funded ZooMu RCN - https://www.nsf.gov/awardsearch/showAward?AWD_ID=2403654&HistoricalAwards=false

**Comment by gparosenberg** [comment]
> The terms lifeStage, reproductiveCondition, and vitality are all attributes of an organism at a place and time - things that can change from one Event (Occurrence) to another. The causeOfDeath term would be singularly tied to the Organism - it is not something that can change from one Event (Occurrence) to another.
> 
> The term individualID no longer exists in Darwin Core.

If we are talking about things that are observed only once, then it is true that it is tied to the organism. But supposed there multiple observations of the same individual. It might be alive on the first observation and dead on the second one. Or there can be multiple observations post mortem, for example, of the time course of decay. If there is no individualID in the Darwin core does that mean that each observation stands on its own?

**Comment by tucotuco** [comment]
dwc:individualID was deprecated in favor or dwc:orgnismID, when the dwc:Organism class came into being, so the concept still exists.

**Comment by AugustT** [comment]
+1

**Comment by tucotuco** [comment]
Having detected concurrence with the proposal to have this term organized in the dwc:Organism class, I made that change to the original proposal in the first comment.

Now, the proposed usage comment is, "Intended to be used only when the value in dwc:vitality indicates that a dwc:Organism was dead at the moment of observation."  Is that restriction really necessary? Couldn't one populate causeOfDeath without populating dwc:vitality?  In fact, in a structured context for Darwin Core in which a dwc:Organism class has unchanging properties while dwc:Occurrence has observations of properties of a dwc:Organism that can change from one dwc:Occurrence to another (such as dwc:lifeStage, dwc:reproductiveCondition, dwc:behavior, and dwc:vitality), this usage comment would be in conflict. Imagine this scenario, a dwc:Organism is live captured and banded. Later it died by hitting a window and was collected. The dwc:Organism should now rightfully have a non-empty causeOfDeath, but there is at least one dwc:Occurrence record for the dwc:Organism that doesn't meet the criteria in the proposed usage comments. I would leave out the usage comments to avoid this semantically problematic scenario.

**Comment by tucotuco** [comment]
There is strong support for the addition of this term organized within the dwc:Organism class. However, the usage comment as proposed ("Intended to be used only when the value in dwc:vitality indicates that a dwc:Organism was dead at the moment of observation.") is anticipated to be problematic. The last sentence of the proposed definition is actually more apt as a usage comment, so I have taken the liberty of moving it there. The proposal can proceed without the original usage comment, which can be discussed and proposed in a future term change issue. 

Though I haven't made changes to the examples, they should be carefully considered if a controlled vocabulary is made. The definition says this is a cause, so the values for it should be causes, which then have to be nouns or noun phrases (e.g., `trap`; `poison`; `starvation`; `drowning`; `shooting`; `old age`; `vehicle collision`; `disease`; `herbicide`; `burning`; `infanticide`) rather than verbs.

The following consensus version of the proposal will be promoted to the Executive Review phase.

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): causeOfDeath
* Term label (English, not normative): Cause Of Death
* Organized in Class (e.g., Occurrence, Event, Location, Taxon): Organism
* Definition of the term (normative): An indication of the known or suspected cause of death of a dwc:Organism. 
* Usage comments (recommendations regarding content, etc., not normative): The cause may be due to natural causes (e.g., disease, predation), human-related activities (e.g., roadkill, pollution), or other environmental factors (e.g., extreme weather events).
* Examples (not normative): `trapped`; `poisoned`; `starved`; `drowned`; `shot`; `old age`; `roadkill`; `disease`; `herbicide`; `burned`; `infanticide`
* Refines (identifier of the broader term this term refines; normative): none
* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): none
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative): none
